src/app/[locale]/settings/_components/mnestix-connections/MnestixConnectionServerActions.tsx

```diff
@@ -0,0 +1,17 @@
+'use server';
+import { prisma } from 'lib/database/prisma';
+import { ConnectionType } from '@prisma/client';
+
+export async function getConnectionDataAction() {
+    return prisma?.mnestixConnection.findMany({ include: { type: true } });
+}
+
+export async function getConnectionDataByTypeAction(type: ConnectionType) {
```

Comment:
Maybe we should move this file to a shared folder if we use it from different pages

Reply:
-> I'll move the file in an upcoming subtask, otherwise mergin could get difficult